rpcserver.go

```diff
@@ -533,9 +533,17 @@ func (r *rpcServer) Start() error {
 	// TODO(roasbeef): eventually also allow the sub-servers to themselves
 	// have a REST proxy.
 	mux := proxy.NewServeMux()
+	grpcEndpoint := cfg.RPCListeners[0].String()
```

Comment:
nit: add a comment why this string replacing is needed.

Reply:
Done.